### PR TITLE
Gross median weekly pay

### PR DESCRIPTION
**Pull request by santhosh-thangavel**
changed mapping to new measures
Code changes in transformation

#8

### CODE REVIEW

File: gross-median-weekly-pay/gross-median-weekly-pay.json [hunk 1]
```diff
@@ -1,56 +1,54 @@
...
-    "description": "Average (median) weekly pay, based on where people live",
-    "license": "http://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/",
-    "keywords": [
-        "subnational",
-        "income",
-        "united-kingdom"
-    ],
-    "publisher": "https://www.gov.uk/government/organisations/office-for-national-statistics",
-    "themes": [
-        "sub-national-data"
```
Comment (sazzadur-ons) [inline gross-median-weekly-pay/gross-median-weekly-pay.json:15]: "themes": [
        "https://www.ons.gov.uk/employmentandlabourmarket/peopleinwork"
    ],

File: gross-median-weekly-pay/gross-median-weekly-pay.json [hunk 2]
```diff
@@ -1,56 +1,54 @@
...
-    ],
-    "columns": {
-        "Period": {
-            "from_template": "year"
-        },
-        "Area": {
-            "from_template": "statistical-geography"
-        },
-        "Mission": {
```
Comment (sazzadur-ons) [inline gross-median-weekly-pay/gross-median-weekly-pay.json:24]: "Mission": false,
"Category": false,
"Indicator": false,

File: gross-median-weekly-pay/gross-median-weekly-pay.json [hunk 3]
```diff
@@ -1,56 +1,54 @@
...
+       "united-kingdom"
+   ],
+   "publisher": "https://www.gov.uk/government/organisations/office-for-national-statistics",
+   "themes": [
+       "sub-national-data"
+   ],
+   "columns": {
+       "Period": {
+           "from_template": "year",
+           "code_list": false
```
Comment (sazzadur-ons) [inline gross-median-weekly-pay/gross-median-weekly-pay.json:20]: Remove "code_list": false line

File: gross-median-weekly-pay/gross-median-weekly-pay.json [hunk 4]
```diff
@@ -1,56 +1,54 @@
...
+       },
+       "Area": {
+           "from_template": "statistical-geography",
+           "code_list": false
```
Comment (sazzadur-ons) [inline gross-median-weekly-pay/gross-median-weekly-pay.json:24]: Remove "code_list": false line

File: gross-median-weekly-pay/gross-median-weekly-pay.json (hunk 1 above)
Comment (santhosh-thangavel) [reply gross-median-weekly-pay/gross-median-weekly-pay.json:15]: Made changes

File: gross-median-weekly-pay/gross-median-weekly-pay.json (hunk 2 above)
Comment (santhosh-thangavel) [reply gross-median-weekly-pay/gross-median-weekly-pay.json:24]: Set to false

File: gross-median-weekly-pay/gross-median-weekly-pay.json (hunk 3 above)
Comment (santhosh-thangavel) [reply gross-median-weekly-pay/gross-median-weekly-pay.json:20]: Removed

File: gross-median-weekly-pay/gross-median-weekly-pay.json (hunk 4 above)
Comment (santhosh-thangavel) [reply gross-median-weekly-pay/gross-median-weekly-pay.json:24]: Removed "code_list": false line

**Review by sazzadur-ons — COMMENTED** [review]
Looks good. Ready to merge

**Comment by sazzadur-ons** [comment]
Ready to merge

**Pull request merged**